pokemon_data.csv

```diff
@@ -0,0 +1,3 @@
+1,bulbasaur
+2,ivysaur
+3,venusaur
```

Comment:
Try to always end a file with a blank line, that in order to keep writing without issues